metabase/metabase
# fix scatter y-axis range

Closes https://github.com/metabase/metabase/issues/41279

### Description

Scatter chart auto y-axis range should be detached from zero.

### How to verify

Create a scatter charts where all values are far away from zero.
Ensure zero is not included in the y-axis range

### Demo

Before
<img width="1720" alt="Screenshot 2024-04-11 at 11 56 28 AM" src="https://github.com/metabase/metabase/assets/14301985/1a50155b-4a90-4111-9847-a49ef76a62dd">


After
<img width="1725" alt="Screenshot 2024-04-11 at 11 56 13 AM" src="https://github.com/metabase/metabase/assets/14301985/f12e760c-9bfc-4a39-a6d2-e6a32ba09f94">


### Checklist
